app/internal/page/mock.go

```diff
@@ -0,0 +1,186 @@
+package page
+
+import (
+	"context"
+	"errors"
+	"io"
+	"net/http"
+	"net/http/httptest"
+
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/sesh"
+
+	"github.com/gorilla/sessions"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/identity"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/localize"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/onelogin"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/place"
+	"github.com/stretchr/testify/mock"
+)
+
+var MockRandom = func(int) string { return "123" }
```

Comment:
I'm not super comfortable with having these in a common package since I prefer tests only to use things in their own package... but it is all just duplicated so I guess I can learn to like it. Maybe one day we can look into auto generating the mocks with https://vektra.github.io/mockery/

Reply:
Mmmm yeah I get where you're coming from but given the narrow scope of the stuff we're mocking it felt like needless repetition. If we get to a point where we need to deviate on the interfaces just hit me up for the grunt work undoing this 😅 

Interesting mocking package there, I'll take a look for future reference.